python/py-pytorch/Portfile

```diff
@@ -172,26 +170,20 @@ if {${name} ne ${subport}} {
 
     configure.ccache    yes
 
-    # ZSTD support broke with 1.2.0-1.12.0. Check with future releases.
     build.env-append \
                     CMAKE_LIBRARY_PATH=${prefix}/lib:${prefix}/lib/libomp \
-                    OpenCV_DIR=${prefix}/libexec/opencv4/cmake \
+                    BUILD_CUSTOM_PROTOBUF=OFF \
                     USE_CCACHE=OFF \
                     USE_CUDA=OFF \
+                    USE_EIGEN_SPARSE=ON \
+                    USE_SYSTEM_EIGEN_INSTALL=ON \
                     USE_GFLAGS=ON \
                     USE_GLOG=ON \
-                    USE_LEVELDB=ON \
                     USE_LITE_PROTO=ON \
-                    USE_LMDB=ON \
-                    USE_METAL=ON \
-                    USE_NCCL=ON \
-                    USE_OPENCV=ON \
+                    USE_NCCL=OFF \
                     USE_OPENMP=ON \
-                    USE_ROCM=OFF \
-                    USE_TBB=ON \
-                    USE_SYSTEM_TBB=ON \
-                    USE_ZMQ=ON \
-                    USE_ZSTD=OFF
+                    USE_RCCL=OFF \
+                    USE_ROCM=OFF
```

Comment:
see warning about XPU during the build, this likely should set `USE_XPU=OFF`

Reply:
Thanks, done.